README.md

```diff
@@ -89,6 +89,29 @@ var tles = provider.GetTles();
 // Alternatively get a specific satellite's TLE
 var issTle = provider.GetTle(25544);
 ```
+## More details on Observe
+
+The Observe method of the Groundstation class may accept a time interval (in UTC) in which to find multiple possible visibility overpasses for a satellite. Here we determine passes for the next 24 hour period, using a time step of 10 seconds.
+```csharp
+var observations = groundStation.Observe(sat, DateTime.UtcNow, DateTime.UtcNow + TimeSpan.FromHours(24), TimeSpan.FromSeconds(10));
+```
+
+The time step provided to use with groundstation.Observe is used to coarsely perform an initial search for the start, end, and time of max elevation of each overpass, which are then determined to a higher resolution using an interval halving approach. The resolution parameter sets the number of decimal places (in seconds) for determined the time above. This allows using a large time step to for finding the overpasses. Here we use a time step of 10 seconds, but determine the time to a resolution of a hundredth of a second.
+```csharp
+var observations = groundStation.Observe(sat, DateTime.UtcNow, DateTime.UtcNow + TimeSpan.FromHours(24), TimeSpan.FromSeconds(10), resolution: 2);
+```
+The default resolution is 3.
+
+The Observe method also provides a minElevation parameter, with a default of 0. E.g.
+```csharp
+observations = groundstation.Observe(sat, DateTime.UtcNow, DateTime.UtcNow + TimeSpan.FromHours(24), TimeSpan.FromSeconds(10), minElevation: Angle.FromDegrees(7.5));
```

Comment:
Also, there isn't a static method `FromDegrees`, it's just `new AngleDegrees(...)`, although I do like the style more so perhaps we should look into migrating to a single `Angle` class and get rid of the `AngleDegrees` wrapper.

Reply:
Whoops. I got confused with another Angle data structure of mine. I'll sort that out.

Any reason why Angle isn't a value type (struct)?

Reply:
No problem. Angle should definitely be a struct. Must have missed changing it when the class was generated.

Reply:
I made the change to Angle for you to consider - I figured better to do it now, so all the changes to the API are in the next version.

Reply:
The changes look good. I struggled wrapping my head around it when I implemented it, and now's a great time to discuss it: do you think it's bad practice to have an implicit cast for any number to a degree angle? Does that make too many assumptions? It would definitely make bugs harder to spot if you're assuming it's degrees.